### PR TITLE
add triu function to the jax.numpy namespace functions

### PR DESCRIPTION
**Pull request by maciejbalawejder**
I added triu function to jax.numpy and previously I got all test passed but when I got the latest commits I get the error and 75% tests passed. It looks like its related to the jax device class(I put screenshots on the discord server in ivy-frontend-tests section). Is it related to the latest version or there's something wrong with the test function?🤔

### CODE REVIEW

**Comment by maciejbalawejder** [comment]
I also added the `ones` function and it passes all the tests

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Review by MarShaikh — CHANGES_REQUESTED** [review]
I'm not sure if your PR is ready for review, but in any case, please resolve the merge conflicts and request for a review if needed. Thanks!

**Comment by vaatsalya123** [comment]
Hey there @maciejbalawejder you have a merge conflict which need's to be resolved before we can start reviewing this pr though like @MarShaikh said once you have fixed the conflict please feel free to request a review from either of us thanks.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
